kuma/operations.go

```diff
@@ -28,9 +33,9 @@ func (h *handler) ApplyOperation(ctx context.Context, op string, id string, del
 	}
 
 	switch op {
-	case cfg.InstallKuma:
+	case cfg.InstallKumav071, cfg.InstallKumav070, cfg.InstallKumav060:
 		go func(hh *handler, ee *Event) {
-			if status, err := hh.installKuma(del); err != nil {
+			if status, err := hh.installKuma(del, operations[op].Properties["version"]); err != nil {
```

Comment:
it would be nice if the key were constants, maybe also centralized?

Reply:
@mgfeller  These are maps because they have are multi-property attributes. Its either map or struct, went for map because its easier to maintain.